ui/DesignSystem/Component/Header/Large.svelte

```diff
@@ -4,7 +4,7 @@
 
   export let style = null;
   export let entity = null;
-  export let variant = null; // profile | org
+  export let variant = null; // profile | project | org
```

Comment:
This component is a little easier to reason about now - after you merge `master` you can just specify the `name`, `urn`, etc.

Reply:
I added more variables now (description, stats) and depended showing the avatar on if an avatar fallback is provided